docs/admin/access.rst

```diff
@@ -89,7 +89,7 @@ Custom
 .. image:: /images/project-access.png
 
 To allow access to this project, you have to add the privilege either
-directly to the given user, or group of users in the Django admin interface,
+directly to the given user, or group of users in the Weblate admin interface,
```

Comment:
Historically, we use Django admin interface. The long-term plan is to replace it for normal management operations by Weblate one, so this is wrong - we still use Django admin interface here.

See https://github.com/WeblateOrg/weblate/issues/2363 for more details.